src/main/java/io/icker/factions/api/persistents/Faction.java

```diff
@@ -224,7 +229,7 @@ public void setHome(Home home) {
     }
 
     public Relationship getRelationship(UUID target) {
-        return relationships.stream()
+        return this.getRelationships().stream()
```

Comment:
I don't think it is necessary to ensure all relationships are valid here because we can assume the faction being supplied does exist.

Reply:
It saves on a little performance to avoid the hash map lookup for each relationship, which is important because this may be run while ensuring interactions are allowed.